### DISCUSSION TITLE
Personalities Over Principals:  The St. Cloud Intergroup Meeting Guide App Feed Has Been Now Been Suspended for One Full Year

### DISCUSSION THREAD

**Discussion by MinnesotaSwanny** [code4recovery/12-step-meeting-list#1146] [Q&A] [answered]
Greetings!

The St. Cloud Intergroup Meeting Guide App Feed has now been suspended for one full year, despite numerous appeals made to the App Support Team, Group Services and other leadership at the General Service Office in New York.  We even codified a new [Meeting Directory Listing Policy](https://aasaintcloud.org/home/meeting-directory-listing-policy/) on 1/7 of this year that we have also shared with the leadership at the GSO. 

We received some high level platitudes about some re-structuring of the technology policies by the General Service Board.  All requests for follow-up and timeframes for change have gone unanswered. @joshreisner - I know this is not normally in your wheelhouse as the application support provider, but given how influential you are - it might help break the log jam if you followed up with App Support about what specific conditions need to be met by the St. Cloud Intergroup for our Meeting Guide App feed to be re-instated?  Is it a time based requirement (ex:  SCI needs to wait 18 months for reinstatement)?  Is it an action we need to take (ex: we show up at the next General Service Board meeting and beg forgiveness or demonstrate some sort of remediation)?  Is it a personality based requirement (ex:  We are waiting for a Trustee Emeriti to give his blessing for the SCI Meeting Guide App feed to be turned back on)?  Is it really a permanent termination of the SCI Meeting Guide App feed but we are calling it a suspension to soften the blow?

Thanks in advance for your consideration on my appeal!

In Service,

Alan

**Comment by joshreisner** [comment, marked as answer]
hi @MinnesotaSwanny thanks for the update. it's a tricky issue. we will talk about it internally at Code for Recovery and i will see if i can get more context from GSO on their thinking - they may very well be in discussions about these policies, and knowing AA, discussions like that can take some time!

at Code for Recovery i believe the question will be more of a technical one - is there some service we can provide or refine to help in situations like this?

in my [earlier comment](https://github.com/code4recovery/spec/discussions/51) i wondered if you might leverage your proximity to nearby groups:

> Have you been in touch with Minneapolis Intergroup or the Area? Could they possibly import these listings from you?

this could have the benefits of resolving discrepancies on that website as well as on the app - have you considered that approach or had any luck there?

**Reply by MinnesotaSwanny** [reply]
Hi @joshreisner!  Thank you for your reply.  Having our sister Intergroup in Minneapolis list meetings for us to get the Meeting Guide App will only work for those meetings where our catchment areas (in terms of the groups we serve) overlay each other.  So we at SCI do not see this as a valid solution.  I also think it is masking the real problem.  As I see it - The App Support Team is being used (unbeknownst to them) to weaponize the Meeting Guide App by keeping the feed from SCI suspended indefinitely to disparage our Intergroup since from one particular past Trustee Emeriti does not want SCI to even exist because the founding groups and members of SCI did not ask his permission (thus, the title of my post).

You can learn more about the history of SCI on this new history [page](https://aasaintcloud.org/home/sci-history/) that we created to tell our story.

Please let me know what you hear back from the App Support Team / GSO. =)

Thanks Josh!

In Service,

Alan
612-865-8810
alanjswanson@hotmail.com

**Reply by MinnesotaSwanny** [reply]
Hi Josh,

I have heard nothing back on when the St. Cloud Intergroup Meeting Guide App feed will be turned back on and what other conditions have to be met.  Have you been able to get any traction with the App Support Team on this?

**Reply by tim-rohrer** [reply]
Hi Alan,

We did inquire about this with our contacts and the issue is being discussed/worked in New York.

C4R provides the open source software for websites to maintain meeting lists. We are not part of the approval process for Meeting Guide registrations; because of this and our traditions, we do not intend to become more involved.

Please direct further queries about this to GSO.

Feel free to reach out to us if you need help with the functioning of the plugin.

-Tim

**Reply by MinnesotaSwanny** [reply]
Hi Tim/Josh,

Thank you for reaching out to your counter parts at the App Support Team at the GSO.  I have heard it said that in AA, "speed kills".  It has been over a year since the St. Cloud Intergroup Meeting Guide App feed was suspended, and no definitive timelines or criteria for reinstatement have been provided.  It is the general feeling of the members of our Intergroup that the Meeting Guide App feed is being weaponized against us.  We thought you should know that in case you have some concerns about your product being used in that sort of way.  Thanks.
